src/controllers/bookings.cr

```diff
@@ -455,7 +456,11 @@ class Bookings < Application
         starting: starting, ending: ending, booking_type: booking_type, user_id: user_id
       )
     query = query.where { id != new_booking.id } if new_booking.id_column.defined?
-    query.to_a
+    query.to_a.reject do |booking|
```

Comment:
I would have preferred to use the PostgreSQL `&&` array operator [Array Functions and Operators](https://www.postgresql.org/docs/9.1/functions-array.html) in the query above, but couldn't get it working.

Reply:
Maybe put a `TODO`?